torch/csrc/jit/serialization/mobile_bytecode.fbs

```diff
@@ -0,0 +1,178 @@
+namespace torch.jit.mobile.serialization;
+
+struct Int {
+  int_val:long;
+}
+
+struct Bool {
+  bool_val:bool;
+}
+
+struct Double{
+  double_val:double;
+}
+
+struct PerTensorAffineSchema {
+  q_scale:double;
+  q_zero_point:int;
+}
+
+table QuantizedSchema {
+  qscheme:byte;
+  scale: double;
+  zero_point: int;
+  scales:TensorMetadata;
+  zero_points:TensorMetadata;
+  axis:int;
+}
+
+table TensorMetadata {
+  // torch._utils _rebuild_tensor_v2
+  storage_location_index: uint;
+  // enum ScalarType
+  scalar_type: byte;
+  storage_offset: int;
+  sizes:[int];
+  strides:[int];
+  requires_grad:bool;
+
+  // only set for quantized tensors
+  quantized_schema:QuantizedSchema;
+}
+
+table String {
+  data: string;
+}
+
+table List {
+  items: [uint];
+  annotation_str: string;  // to recover key/val type
+}
+
+table IntList {
+  items: [long];
+}
+
+table DoubleList {
+  items: [double];
+}
+
+table BoolList {
+  items: [bool];
+}
+
+table Tuple {
+  items: [uint];
+}
+
+table Dict {
+  keys: [uint];
+  values: [uint];
+  annotation_str: string;  // to recover key/val type
+}
+
+enum TypeType : ubyte {
+  UNSET,
+  CLASS_WITH_FIELD,
+  CUSTOM_CLASS,
+  CLASS_WITH_SETSTATE,
+  NON_OBJ,
+}
+
+table ObjectType {
+  type_name:string;
+  type: TypeType;
+  // Below fields are optional
+  attr_names:[string];
+}
+
+table Object {
+  type_index: uint;
+  state: uint;
+  attrs: [uint];
+  setstate_func: uint;
+}
+
+struct Instruction {
+  // Should op be enum instead?
+  op:byte;
+  n:ushort;
+  x:int;
+}
+
+table Operator {
+  name:string;
+  overload_name:string;
+  num_args_serialized:int = -1;
+}
+
+table Arg {
+  name:string;
+  // Why do we use string to represent types
+  // rather than index into Code.types?
+  type: string;
+  default_value:uint;  // position into ivalues
+}
+
+table Schema {
+  arguments:[Arg];
+  returns:[Arg];
+}
+
+table DebugInfo {
+  debug_handle:[long];
+}
+
+table Function {
+  qn:string;
+  instructions:[Instruction];
+  operators:[Operator];
+  constants:[uint];  // index to ivalue
+  type_annotations: [string];
+  register_size:int;
+  schema:Schema;
+  debug_info:DebugInfo;
+}
+
+table StorageData {
+  data: [ubyte] (force_align: 16);
```

Comment:
I remember we don't need force_align anymore right? The reason is we enforce the alignment manually instead of relying on generated API from FB. Maybe put a comment here instead?

Reply:
Turns out, (at least in the version of flatbuffer in BUCK, building for android) (force_align:16) doesnt do anything (i.e. could not find anything different from the generated headers). Now, we might still want to keep this annotation just to document intent.